cypress/e2e/registerPageTests.spec.js

```diff
@@ -0,0 +1,33 @@
+/// <reference types="cypress"/>
+
+import { RegisterPage } from '../pages/RegisterPage.page.js'
+import { HeaderNavigationElements } from '../pages/HeaderNavigationElements.page.js'
+
+describe('Login form test', () => {
+    const registerPage = new RegisterPage()
+    const headerNavigationElements = new HeaderNavigationElements()
+
+    it('inputs validation check', () => { 
+        registerPage.open()
+        registerPage.elements.fulleNameInput().click().type('tes')
```

Comment:
```suggestion
        registerPage.elements.fulleNameInput().type('tes')
```
можна без кліку відразу вводити текст